components/raftstore/src/store/worker/read.rs

```diff
@@ -673,10 +601,7 @@ where
         rd.filter(|r| !r.pending_remove)
     }
 
-    pub fn pre_propose_raft_command(
-        &mut self,
-        req: &RaftCmdRequest,
-    ) -> Result<Option<(D, RequestPolicy)>> {
+    pub fn request_check(&mut self, req: &RaftCmdRequest) -> Result<Option<D>> {
```

Comment:
What does `request_check` mean?

Reply:
Validate request.

Reply:
Bad style, we use `verb + noun` instead of `noun + verb`.

Reply:
Fine.